daemon/src/management/InodeEntry.h

```diff
@@ -7,7 +7,12 @@ struct InodeEntry
 
 struct InodeDirectoryEntry: InodeEntry
 {
-    char* inodeName;
+    // please, do not modify inodeName after adding to directory to avoid conflicting names
+    char* inodeName = nullptr;
```

Comment:
How about adding getter and setter?

Reply:
Getters and setters for this struct are not really necessary. Getter is useless because field is public and setting name outside the directory may cause name duplication inside it, as stated in the comment

Reply:
ok